src/core/connection.c

```diff
@@ -5626,6 +5670,54 @@ QuicConnProcessUdpUnreachable(
     }
 }
 
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+QuicConnProcessRouteCompletion(
+    _In_ QUIC_CONNECTION* Connection,
+    _In_ const uint8_t* PhysicalAddress,
+    _In_ uint8_t PathId,
+    _In_ BOOLEAN Succeeded
+    )
+{
+    uint8_t PathIndex;
+    QUIC_PATH* Path = QuicConnGetPathByID(Connection, PathId, &PathIndex);
+    if (Path != NULL) {
+        if (Succeeded) {
+            if (Path->IsActive) {
```

Comment:
We shouldn't have this `if` just for the comment below. Please reorganize things so it's not necessary.

Reply:
Oh, I forgot to clean this up.